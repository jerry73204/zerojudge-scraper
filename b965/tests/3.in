8 6 3
3 2 7 2 6 4
2 1 4 3 2 5
0 7 7 9 2 1
8 0 8 4 2 1
1 8 5 3 8 4
4 2 4 0 1 8
7 1 1 8 7 3
0 9 0 7 5 0
0 1 0