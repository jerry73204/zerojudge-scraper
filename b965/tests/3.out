8 6
0 9 0 7 5 0
7 1 1 8 7 3
4 2 4 0 1 8
1 8 5 3 8 4
8 0 8 4 2 1
0 7 7 9 2 1
2 1 4 3 2 5
3 2 7 2 6 4